### ISSUE TITLE
Format question asked twice when initializing configuration

### ISSUE THREAD

**Issue by 7ma7X** [eslint/eslint#11617]
**Tell us about your environment**

* **ESLint Version:**

5.16.0

* **Node Version:**

10.15.0

* **npm Version:**

6.8.0

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

When running config initialization wizard, `What format do you want your config file to be in?` appeared twice. This seems to be unnecessary.

<img width="1028" alt="スクリーンショット 2019-04-14 13 12 43" src="https://user-images.githubusercontent.com/36184621/56088346-e080aa00-5eb9-11e9-9d61-feafeca5c6b4.png">

**What did you expect to happen?**

The latter question should not appear.

**What actually happened? Please include the actual, raw output from ESLint.**

Format question asked twice

**Are you willing to submit a pull request to fix this bug?**

Yes
